### PR TITLE
Accessibility

### PR DESCRIPTION
**Pull request by skcain**
To assessment inputs

### CODE REVIEW

File: views/assessment.rb [hunk 1]
```diff
@@ -9,33 +9,42 @@ def images
           { trait: 'Following instructions',
             reverse_trait: 'Self-starter',
             title: 'Following instructions',
+            alt: 'Person is looking at a recipe on a computer before they begins chopping vegetables',
```
Comment (antislice) [inline views/assessment.rb]: extra `s` on begins

File: views/assessment.rb [hunk 2]
```diff
@@ -9,33 +9,42 @@ def images
...
             file: 'instructions.jpg' },
           { trait: 'Working with your hands',
             reverse_trait: 'Technical',
             title: 'Working with your hands',
+            alt: 'Person has a a screwdriver and wrench in their hands and is working on mechanical parts',
```
Comment (antislice) [inline views/assessment.rb]: extra `a`: Person has a a screwdriver

File: views/assessment.rb [hunk 3]
```diff
@@ -9,33 +9,42 @@ def images
...
             title: 'Following a routine every day',
+            alt: 'Person is driving a bus',
             file: 'routine.jpg' },
           { trait: 'Organized',
             title: 'Being well-organized',
+            alt: 'Picture of a well-organized closet',
             file: 'organized.jpg' },
           { trait: 'No fear of heights',
             title: 'Comfortable with heights',
+            alt: 'Person is working on powerlines several feet off the ground',
```
Comment (antislice) [inline views/assessment.rb]: Is there a space in 'powerlines'?

**Review by pacarole — APPROVED** [review]

**Review by antislice — CHANGES_REQUESTED** [review]
I have suuuuuper minor comments. Otherwise I don't know enough about accessibility to say if there's anything missing.

File: views/assessment.rb (hunk 3 above)
Comment (skcain) [reply views/assessment.rb]: Yes! Good catch.

File: views/assessment.rb (hunk 2 above)
Comment (skcain) [reply views/assessment.rb]: 🎣

File: views/assessment.rb (hunk 1 above)
Comment (skcain) [reply views/assessment.rb]: 🎣

**Review by antislice — APPROVED** [review]

**Pull request merged**